deploy/service_account.yaml

```diff
@@ -32,3 +32,63 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: resultserver
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: co-metrics-client
+rules:
+  - nonResourceURLs:
+      - /metrics
+      - /metrics-co
+    verbs:
+      - get
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: co-metrics-client
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: co-metrics-client
+subjects:
+  - kind: ServiceAccount
+    name: default
```

Comment:
Why are we using the default SA? Is it so that anybody could scrape the metrics?

Reply:
It's for the manual CLI testing with `oc run`. Running the workload to scrape the metrics would use default, and use the token specified below.

It's probably better to not have this permission set enabled on deployment, but let it be created by the CI harness for its use in CI.